Gemfile.lock

```diff
@@ -184,25 +184,25 @@ GEM
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.6)
+    tzinfo (1.2.7)
       thread_safe (~> 0.1)
     uglifier (4.2.0)
       execjs (>= 0.3.0, < 3)
-    web-console (4.0.1)
+    web-console (4.0.4)
       actionview (>= 6.0.0)
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webdrivers (4.2.0)
+    webdrivers (4.4.1)
       nokogiri (~> 1.6)
       rubyzip (>= 1.3.0)
       selenium-webdriver (>= 3.0, < 4.0)
-    websocket-driver (0.7.1)
+    websocket-driver (0.7.3)
```

Comment:
### [websocket-driver](https://github.com/faye/websocket-driver-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.7.1 → 0.7.3

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/faye/websocket-driver-ruby/compare/0.7.1...0.7.3).

These are the individual commits:
- (e568366) [Formatting change: {...} should have spaces inside the brace…](https://github.com/faye/websocket-driver-ruby/commit/e568366f82ffd284208ad7b0081e09041e3354ec)
- (d05eb72) [Mention license change in the changelog](https://github.com/faye/websocket-driver-ruby/commit/d05eb72d2d815d4e1b54a115a8fd4c18606ef74a)
- (b00f587) [Emit ping and pong events from the Server driver](https://github.com/faye/websocket-driver-ruby/commit/b00f58701ec008145d5075075d12271b762381db)
- (202e180) [Allow rack.input.read to return a frozen string](https://github.com/faye/websocket-driver-ruby/commit/202e1800e0e948d83ec18368bdc6cd4651c36eb0)
- (db09a1a) [Update Ruby versions 2.4 to 2.7 on Travis](https://github.com/faye/websocket-driver-ruby/commit/db09a1af222e504ae85e9947cc8170fddedccd7a)
- (46aafa4) [Set {source,target}_version to work with the current version…](https://github.com/faye/websocket-driver-ruby/commit/46aafa4e9963c344775318e7a646fb4f6b9e7d29)
- (f882bc5) [Bump version to 0.7.2](https://github.com/faye/websocket-driver-ruby/commit/f882bc56d530564614c5ad81e61eb99c445bd99a)
- (e87bfef) [Accept HTTP responses that have an empty reason-phrase, per …](https://github.com/faye/websocket-driver-ruby/commit/e87bfef9bc0013e32546f2b8d158d20212915076)
- (eb643f5) [Bump version to 0.7.3](https://github.com/faye/websocket-driver-ruby/commit/eb643f59499dec53939b30eebba5ca2f5e7296a7)


</details>